app/routes/tasks.js

```diff
@@ -4,21 +4,18 @@ import { inject as service } from '@ember/service';
 
 export default class TasksRoute extends Route {
   @service store;
-  @service dataLoader;
+  @service('requests/task/task-service') requestsTaskTaskService;
```

Comment:
Fixed here: https://github.com/Mr-Smyth/task-manager/pull/18/commits/45d8ab87eea8e95f8870e69e6f46317d2e2ae701

Reply:
> naming convention should be maintained

That is true. However, if we want to stay according to the conventions, IMO we should not have created subdirectories in `services` directory. 
Ember keeps directory tree by `functionality` e.g. controllers, services, models etc.
Another approach is to keep order by `domain` so we should have: users, tasks etc. as main directories.
Here we actually mixed both because under services we have domain specific subdirectory. 
Honestly, I am totally fine with that - but I believe this is the reason we have to “break” slightly this naming convention.
Nonetheless, good shout !